iam/role/role.tf

```diff
@@ -24,11 +25,25 @@ variable "role_description" {
   default     = "Terraform created role"
 }
 
+locals {
+  trusted_accounts_size = "${length( var.trusted_accounts )}"
+  users_size            = "${length( var.users )}"
+  arn_list_size         = "${local.accounts_size * local.users_size}"
+}
+
 # ------------------------------------------------------------------------------
 # Resources
 # ------------------------------------------------------------------------------
 data "aws_iam_account_alias" "current" {}
 
+data "null_data_source" "users" {
+  count = "${local.arn_list_size}"
+
+  inputs = {
+    users = "${format("arn:aws:iam::%s:user/%s", var.trusted_accounts[count.index % local.accounts_size], var.users[count.index / local.accounts_size])}"
```

Comment:
Thats the stuff! Looks about right! 🤖

Reply:
Nice!

Reply:
Hehe took me a while to figure out if this actually was correct, I need to program more :)